editor-components/feature/feature6/feature6.tsx

```diff
@@ -121,6 +110,37 @@ class Feature6 extends BaseFeature {
       key: "itemCount",
       displayer: "Item count in a row",
       value: 3,
+      max: 3,
+    });
+    
+    let button: TypeUsableComponentProps = {
+      type: "object",
+      key: "button",
+      displayer: "Button",
+      value: [
+        {
+          type: "string",
+          key: "buttonText",
+          displayer: "Button Text",
+          value: "VIEW ALL CASE STUDIES",
+        },
+        {
+          type: "page",
+          key: "url",
+          displayer: "Button Link",
+          value: "",
+        },
+      ],
+    };
+
+    this.addProp({
+      type: "array",
+      key: "buttons",
+      displayer: "Buttons",
+      value: [JSON.parse(JSON.stringify(button))],
+      additionalParams: {
+        maxElementCount:  2
+      }
```

Comment:
Can you explain this section? Why did you choose this way? And why do you need it?